site/_docs/reference.md

```diff
@@ -2630,9 +2630,9 @@ LIMIT 10;
 
 Result
 
-| c1     | c2    | c3      | c4      |
-| ------ | ----- | ------- | ------- |
-| OBJECT | ARRAY | INTEGER | BOOLEAN |
+|    c1    |   c2    |    c3     |    c4     |
```

Comment:
There are a lot of white-space changes in this file which do not seem necessary. It is better to avoid modifying lines if it is not needed to avoid cluttering the git history. If things are reasonably readable after adding `:` I would suggest to keep the rest as they were before.

Reply:
I agree with your comments and finished the changes.